### PR TITLE
add_comment_feature&modify_about_me

### PR DESCRIPTION
**Pull request by pikerple**

### CODE REVIEW

**Review by layog — CHANGES_REQUESTED** [review]
Hi Mandy,

In your previous PR, you are calling `/new-task` when adding a new comment. I think you should setup another endpoint for comments and not use the same task list. Sorry I missed this point in the previous PR, my bad!

**Comment by pikerple** [comment]
I've modify it and add week3 libraries feature.

File: portfolio/src/main/java/com/google/sps/servlets/Comment.java [hunk 1]
```diff
@@ -12,18 +12,18 @@
...
-    this.timestamp = timestamp;
-  }
+    public Comment(String m, String i, long t){
+        message = m;
+        imageUrl = i;
+        timestamp = t;
+    }
+
+    String message;
+	String imageUrl;
```
Comment (layog) [inline portfolio/src/main/java/com/google/sps/servlets/Comment.java:27]: nit: Formatting error

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,43 @@
...
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Mandy!</h1>");
-  }
-}
+    // query comments from database
+		Query query = new Query("Comments").addSort("timestamp", SortDirection.ASCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+		PreparedQuery results = datastore.prepare(query);
+		
+		// Put the comments into a list in order to use gson to convert
```
Comment (layog) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: nit: Formatting

**Review by layog — APPROVED** [review]
Hi Mandy,

There are some formatting errors, but I am not going to block the merging as this is the last week and you might not have time to work on this again.

**Pull request merged**